elasticdl/python/master/optimizer_wrapper.py

```diff
@@ -262,6 +262,7 @@ def _generate_lookup_keys(self, grads_and_vars):
             # de-duplicate gradient's indices
             unique_ids, indices = tf.unique(grad.indices)
             unique_ids = unique_ids.numpy()
+            # TODO: support grads_and_vars with duplicated layer name
```

Comment:
Is this going to be fixed soon? If not, log a warning when detected?

Reply:
Added.